src/pages/workspace/expensifyCard/issueNew/ConfirmationStep.tsx

```diff
@@ -122,11 +125,31 @@ function ConfirmationStep({policyID, backTo}: ConfirmationStepProps) {
                         isAlertVisible={!!errorMessage}
                         isDisabled={isOffline}
                         isLoading={issueNewCard?.isLoading}
-                        onSubmit={submit}
+                        onSubmit={() => setIsValidateCodeActionModalVisible(true)}
                         buttonText={translate('workspace.card.issueCard')}
                     />
                 </View>
             </ScrollView>
+            {!!issueNewCard && (
+                <ValidateCodeActionModal
+                    handleSubmitForm={submit}
+                    isLoading={issueNewCard?.isLoading}
+                    sendValidateCode={() => User.requestValidateCodeAction()}
```

Comment:
Let's add `hasMagicCodeBeenSent` props here to make it consistent with other places.

Reply:
<img width="967" alt="Screenshot 2024-11-13 at 15 08 49" src="https://github.com/user-attachments/assets/af037ab8-91e4-4beb-84aa-54f1075fea0e">

@hungvu193 i updated